modules/cco-ccoctl-creating-at-once.adoc

```diff
@@ -171,13 +171,13 @@ You can verify that the RAM users and policies are created by querying Alibaba C
 +
 [source,terminal]
 ----
-$ cp ./<path_to_ccoctl_output_dir>/manifests/*credentials.yaml ./<path_to_installation>dir>/manifests/
+$ cp ./<path_to_ccoctl_output_dir>/manifests/*credentials.yaml ./<path_to_installation_dir>/manifests/
```

Comment:
Thank @mtulio ! So to confirm, In step 6 `<path_to_ccoctl_output_dir>` should be updated to `<path_to_installation_dir>` and `*credentials.yaml` should be updated to `*credentials*.yaml`?

Reply:
@jldohmann Let's keep that typo on the radar, for now, we need to confirm with Hive engineer if there are steps missing, I did not follow recent changes on the component, as mentioned on Slack. So they will the best people to move it forward.

Reply:
> The ./<path_to_ccoctl_output_dir> path was not referenced in any moment to extract the credentialRequests. The manifests will be extracted, step 5, in the same directory with the command that was executed.

I can see on line 122, we have mentioned to extract credentials requests to <path_to_ccoctl_output_dir> 

> The command cp *credentials.yaml will not cover all CredentialRequests extracted from step 5.:

instead of `cp ./<path_to_ccoctl_output_dir>/manifests/*credentials.yaml ./<path_to_installation_dir>/manifests/` we should use command ` cp ./<path_to_ccoctl_output_dir>/manifests/* ./<path_to_installation_dir>/manifests/`. All the files generated by ccoctl in <path_to_ccoctl_output_dir> will be credentials secrets.

Reply:
> > The ./<path_to_ccoctl_output_dir> path was not referenced in any moment to extract the credentialRequests. The manifests will be extracted, step 5, in the same directory with the command that was executed.
> 
> I can see on line 122, we have mentioned to extract credentials requests to <path_to_ccoctl_output_dir>

Yes, there's on the source, but seems not to be rendered correctly. Look at the preview:
https://deploy-preview-44248--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_aws/manually-creating-iam.html#manually-create-iam_manually-creating-iam-aws

I would say that part should be there to effectively create the credentials and it's not included on the rendered page/steps:
```
$ ccoctl aws create-all --name=<name> --region=<aws_region> --credentials-requests-dir=<path_to_directory_with_list_of_credentials_requests>/credrequests
```

Note: that command, for AWS, seems not to add the flag `--output-dir` with value of `path_to_ccoctl_output_dir`

Reply:
I would say we go ahead and add --output-dir parameter to the command. So the full command will be 

```
$ ccoctl aws create-all --name=<name> --region=<aws_region> --credentials-requests-dir=<path_to_directory_with_list_of_credentials_requests>/credrequests --output-dir=<path_to_ccoctl_output_dir>
```

Reply:
I think there may be some confusion here...the deploy preview is just for the `manually-create-identity-access-management.adoc` file, that was the one that was filed in BZ as having issues. 

The `cco-ccotl-creating-at-once.adoc` file can be seen rendered [here](https://docs.openshift.com/container-platform/4.10/authentication/managing_cloud_provider_credentials/cco-mode-sts.html#cco-ccoctl-creating-at-once_cco-mode-sts) for the AWS case; I happened to notice it had a typo and fixed that as well, hence its inclusion in this PR. Does this procedure need updating as well?

Reply:
@akhil-rane @mtulio do y'all mind taking a look at [this link to the rendered AWS case ](https://docs.openshift.com/container-platform/4.10/authentication/managing_cloud_provider_credentials/cco-mode-sts.html#cco-ccoctl-creating-at-once_cco-mode-sts)to check if it has issues (see above commment). Apologies for the confusion, thank you

Reply:
@jldohmann Documentation reviewed. Please take a look at the issues I found: https://docs.google.com/document/d/1ifa8SGF7aAdxtKboZmP1lStHwkBnPqBSjDWTIopQ-5g/edit#

/cc @akhil-rane - also let me know.